shell/browser/api/electron_api_app.cc

```diff
@@ -901,6 +903,20 @@ bool App::IsPackaged() {
 
 #if BUILDFLAG(IS_WIN)
   return base_name != FILE_PATH_LITERAL("electron.exe");
+#elif BUILDFLAG(IS_MAC)
+  static const base::FilePath::StringType default_helper =
+      "electron helper" + base::ToLowerASCII(content::kMacHelperSuffix_default);
+  static const base::FilePath::StringType renderer_helper =
+      "electron helper" +
+      base::ToLowerASCII(content::kMacHelperSuffix_renderer);
+  static const base::FilePath::StringType plugin_helper =
+      "electron helper" + base::ToLowerASCII(content::kMacHelperSuffix_plugin);
```

Comment:
These need to be [`base::NoDestructor<std::string>`](https://chromium.googlesource.com/chromium/src/+/main/base/no_destructor.h#18) since static objects with nontrivial destructors [aren't allowed](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables).